packages/host/app/services/command-service.ts

```diff
@@ -100,11 +117,13 @@ export default class CommandService extends Service {
   }
 
   //TODO: Convert to non-EC async method after fixing CS-6987
-  run = task(async (command: CommandCard, roomId: string) => {
+  run = task(async (command: MessageCommand, roomId: string) => {
```

Comment:
this being a task really doesnt buy us anything since the service lives forever (the async never needs to be cancelled)

Reply:
Does the caveat about CS-6987 still apply?

Reply:
Actually, looks like we are using derived state from the task in room.gts.